.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Pypi Publishing
+
```

Comment:
I guess this is the file for software release?

Reply:
Exactly!